lib/src/value/color.ts

```diff
@@ -20,6 +20,13 @@ interface HslColor {
   alpha: number;
 }
 
+interface HwbColor {
+  hue: number;
+  whiteness: number;
+  blackness: number;
+  alpha: number;
```

Comment:
shouldn't the alpha channel be optional in those types ? The constructor has a check `color.alpha === undefined`